packages/www/pages/app/stream/[id].tsx

```diff
@@ -193,6 +213,21 @@ const ID = () => {
   const isAdmin = query.admin === "true";
   const tabs = isAdmin ? getTabsAdmin(2) : getTabs(0);
   const backLink = isAdmin ? "/app/admin/streams" : "/app/user";
+  const _stream = stream || {};
+  let { broadcasterHost, region } = _stream;
+  if (!broadcasterHost && lastSession && lastSession.broadcasterHost) {
+    broadcasterHost = lastSession.broadcasterHost;
+  }
+  if (!region && lastSession && lastSession.region) {
+    region = lastSession.region;
+  }
+  let broadcasterPlaybackUrl;
+  const domain = isStaging() ? "monster" : "com";
+  if (stream && stream.region && stream.broadcasterHost) {
+    broadcasterPlaybackUrl = `https://${stream.region}.livepeer.${domain}/stream/${stream.id}.m3u8`;
+  } else if (lastSession && lastSession.region) {
+    broadcasterPlaybackUrl = `https://${lastSession.region}.livepeer.${domain}/stream/${stream.playbackId}.m3u8`;
+  }
```

Comment:
Hmmmmm. It seems like there should be a better way to do this... it would be pretty useful for us to return actual playback URLs from the API. Not something we have to do now but something to keep thinking about.